core/src/main/scala/org/apache/spark/rdd/SqlNewHadoopRDD.scala

```diff
@@ -96,6 +96,11 @@ private[spark] class SqlNewHadoopRDD[V: ClassTag](
 
   @transient protected val jobId = new JobID(jobTrackerId, id)
 
+  // If true, enable using the custom RecordReader for parquet. This only works for
+  // a subset of the types (no complex types).
+  protected val enableUnsafeRowParquetReader: Boolean =
+      sc.conf.getBoolean("spark.parquet.enableUnsafeRowRecordReader", true)
```

Comment:
Sorry, I said this on the wrong PR, but we should get this from the SQLConf since the Spark conf is immutable.